modular_skyrat/code/modules/cargo/packs/bloat.dm

```diff
@@ -21,6 +21,22 @@
 					/obj/item/gun/energy/e_gun/stun)
 	crate_name = "swat taser crate"
 
+/datum/supply_pack/security/armory/swat_tactical
+	name = "SWAT Tactical Operations crate"
+	desc = "Contains two sets of lightweight equipment for tactical operations. Each set contains a vest, night vision helmet, mask, combat belt, and combat gloves. Requires Armory access to open."
+	cost = 6000
+	contains = list(/obj/item/clothing/head/helmet/advanced,
+					/obj/item/clothing/head/helmet/advanced,
+					/obj/item/clothing/suit/armor/vest/advanced,
+					/obj/item/clothing/suit/armor/vest/advanced,
```

Comment:
should be the tech armor instead IMHO

Reply:
or maybe make swat tech-armor a separate crate

Reply:
I dont know if fullbody armor is suitable for lightweight equipment, besides, this was supposed to be a replacement for that node

Reply:
I mean - Fair enough tbh, but i do feel that tech armor MK2 should be accessible somehow, i do think a separate crate with another name can work.